### PR TITLE
Account display name `servername (username) -> username@servername`

### PR DESCRIPTION
**Pull request by TheOneRing**
Issue: #8104

# New:
image

### CODE REVIEW

File: src/libsync/account.cpp [hunk 1]
```diff
@@ -114,13 +114,11 @@ QString Account::displayName() const
...
-    int port = url().port();
+    const int port = url().port();
     if (port > 0 && port != 80 && port != 443) {
-        host.append(QLatin1Char(':'));
-        host.append(QString::number(port));
+        host += QStringLiteral(":%1").arg(QString::number(port));
     }
-    QString dn = QStringLiteral("%1 (%2)").arg(host, user);
-    return dn;
+    return tr("%1@%2").arg(user, host);
```
Comment (TheOneRing) [inline src/libsync/account.cpp:121]: I switched to a translatable string even so that syntax now should be pretty universal

**Review by michaelstingl — APPROVED** [review]
Changelog 👍 
(can't judge the actual code)

**Review by pmaier1 — APPROVED** [review]
<3 (can't judge on code :x)

File: src/libsync/account.cpp (hunk 1 above)
Comment (dschmidt) [reply src/libsync/account.cpp:121]: Yeah, not sure this brings actual benefit ...

**Review by dschmidt — APPROVED** [review]

**Comment by TheOneRing** [comment]
yes the previous string probably needed translations for right to left languages

**Comment by TheOneRing** [comment]
but mail address like things probably not xD

**Pull request merged**